src/util-lua-dns.c

```diff
@@ -165,7 +165,7 @@ static int DnsGetQueryTable(lua_State *luastate)
                     SCFree(c);
                     return LuaCallbackError(luastate, "invalid length");
                 }
-                lua_pushstring(luastate, "rrname");
+                lua_pushstring(luastate, "name");
```

Comment:
It seems the RFC is using qname for name in the query. So we could use this. As a side effect it will avoid ambiguity  with the 'name' in the response.